🐛 Incorrect Manifest Path for Releases

Logs from Prow release Job.

```
Step #0: make set-manifest-image MANIFEST_IMG=gcr.io/k8s-staging-cluster-api-nested/cluster-api-nested-controller MANIFEST_TAG=v20210608-e593785 TARGET_RESOURCE="./config/manager/manager_image_patch.yaml"
Step #0: make[3]: Entering directory '/workspace'
Step #0: fatal: not a git repository (or any parent up to mount point /)
Step #0: Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
Step #0: Updating kustomize image patch file for manager resource
Step #0: sed -i'' -e 's@image: .*@image: '"gcr.io/k8s-staging-cluster-api-nested/cluster-api-nested-controller:v20210608-e593785"'@' ./config/manager/manager_image_patch.yaml
Step #0: make[3]: Leaving directory '/workspace'
Step #0: make[2]: Leaving directory '/workspace'
Step #0: sed: ./config/manager/manager_image_patch.yaml: No such file or directory
Step #0: make[3]: *** [Makefile:300: set-manifest-image] Error 1
Step #0: make[2]: *** [Makefile:289: docker-push-core-manifest] Error 2
Step #0: make[1]: *** [Makefile:268: docker-push-all] Error 2
Step #0: make[1]: Leaving directory '/workspace'
Step #0: make: *** [Makefile:346: release-staging] Error 2
Finished Step #0
ERROR
ERROR: build step 0 "gcr.io/k8s-testimages/gcb-docker-gcloud:v20200619-68869a4" failed: step exited with non-zero status: 2
```

--- a/Makefile
+++ b/Makefile
@@ -286,8 +286,8 @@ docker-push-core-manifest: ## Push the fat manifest docker image for the core im
 	docker manifest create --amend $(CONTROLLER_IMG):$(TAG) $(shell echo $(ALL_ARCH) | sed -e "s~[^ ]*~$(CONTROLLER_IMG)\-&:$(TAG)~g")
 	@for arch in $(ALL_ARCH); do docker manifest annotate --arch $${arch} ${CONTROLLER_IMG}:${TAG} ${CONTROLLER_IMG}-$${arch}:${TAG}; done
 	docker manifest push --purge $(CONTROLLER_IMG):$(TAG)
-	$(MAKE) set-manifest-image MANIFEST_IMG=$(CONTROLLER_IMG) MANIFEST_TAG=$(TAG) TARGET_RESOURCE="./config/manager/manager_image_patch.yaml"
-	$(MAKE) set-manifest-pull-policy TARGET_RESOURCE="./config/manager/manager_pull_policy.yaml"
+	$(MAKE) set-manifest-image MANIFEST_IMG=$(CONTROLLER_IMG) MANIFEST_TAG=$(TAG) TARGET_RESOURCE="./config/default/manager_image_patch.yaml"
+	$(MAKE) set-manifest-pull-policy TARGET_RESOURCE="./config/default/manager_pull_policy.yaml"
 
 .PHONY: set-manifest-pull-policy
 set-manifest-pull-policy:
@@ -319,8 +319,8 @@ release: clean-release ## Builds and push container images using the latest git 
 	# Set the core manifest image to the production bucket.
 	$(MAKE) set-manifest-image \
 		MANIFEST_IMG=$(PROD_REGISTRY)/$(IMAGE_NAME) MANIFEST_TAG=$(RELEASE_TAG) \
-		TARGET_RESOURCE="./config/manager/manager_image_patch.yaml"
-	$(MAKE) set-manifest-pull-policy PULL_POLICY=IfNotPresent TARGET_RESOURCE="./config/manager/manager_pull_policy.yaml"
+		TARGET_RESOURCE="./config/default/manager_image_patch.yaml"
+	$(MAKE) set-manifest-pull-policy PULL_POLICY=IfNotPresent TARGET_RESOURCE="./config/default/manager_pull_policy.yaml"
 	## Build the manifests
 	$(MAKE) release-manifests clean-release-git
 	## Build the development manifests

